app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,95 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.ProductNotFoundException;
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+/**
+ * 상품과 관련된 비즈니스 로직을 담당합니다.
+ *
+ * @see Product
+ * @see ProductRepository
+ * @see com.codesoom.assignment.infra.InMemoryProductRepository
+ * @see com.codesoom.assignment.infra.JpaProductRepository
+ */
+@Service
+@Transactional
+public class ProductService {
+
+    private final ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    /**
+     * 모든 상품을 리턴합니다.
+     */
+    public List<Product> getProducts() {
+        return productRepository.findAll();
+    }
+
+    /**
+     * 주어진 id에 해당하는 상품을 리턴합니다.
+     *
+     * @param id 찾는 상품에 해당하는 id
```

Comment:
만약 `id`가 두 번 반복되는 것이 마음에 들지 않는다면 다음과 같이 하는 방법도 있습니다.
```suggestion
     * @param id 상품의 식별자
```